lnwallet/channel_test.go

```diff
@@ -8658,223 +8658,6 @@ type heights struct {
 	remoteRemove uint64
 }
 
-// TestProcessFeeUpdate tests the applying of fee updates and mutation of
```

Comment:
hmm but we need to test it tho

Reply:
It's a one liner, does it really need a test? Note that this commit series removes a lot of the complexity that is being tested.